dynawo/sources/Models/Modelica/Dynawo/Examples/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
When adding new files to the OS repo, it is better to update the copyright to 2015-2020.

Reply:
sorry, I will correct, nice catch!